site/content/en/docs/Installation/_index.md

```diff
@@ -96,46 +96,89 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
+{{% feature expiryVersion="1.1.0" %}}
 ```bash
 gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.12 \
   --tags=game-server \
   --scopes=gke-default \
   --num-nodes=4 \
   --machine-type=n1-standard-4
 ```
+{{% /feature %}}
+{{% feature publishVersion="1.1.0" %}}
+```bash
+gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.12 \
+  --tags=game-server \
+  --scopes=gke-default \
+  --num-nodes=4 \
+  --no-enable-autoupgrade \
+  --machine-type=n1-standard-4
+```
+{{% /feature %}}
 
 Flag explanations:
 
 * cluster-version: Agones requires Kubernetes version 1.12.
 * tags: Defines the tags that will be attached to new nodes in the cluster. This is to grant access through ports via the firewall created in the next step.
 * scopes: Defines the Oauth scopes required by the nodes.
 * num-nodes: The number of nodes to be created in each of the cluster's zones. Default: 4. Depending on the needs of your game, this parameter should be adjusted.
+{{% feature publishVersion="1.1.0" %}}
+* no-enable-autoupgrade: Disable automatic upgrades for nodes to reduce the liklihood of in-use games being disrupted.
+{{% /feature %}}
 * machine-type: The type of machine to use for nodes. Default: n1-standard-4. Depending on the needs of your game, you may wish to [have smaller or larger machines](https://cloud.google.com/compute/docs/machine-types).
 
 _Optional_: Create a dedicated node pool for the Agones controllers. If you choose to skip this step, the Agones
 controllers will share the default node pool with your game servers which is fine for kicking the tires but is not
 recommended for a production deployment.
 
+{{% feature expiryVersion="1.1.0" %}}
+```bash
+gcloud container node-pools create agones-system \
+  --cluster=[CLUSTER_NAME] \
+  --node-taints agones.dev/agones-system=true:NoExecute \
+  --node-labels agones.dev/agones-system=true \
+  --num-nodes=1
+```
+{{% /feature %}}
+{{% feature publishVersion="1.1.0" %}}
 ```bash
 gcloud container node-pools create agones-system \
   --cluster=[CLUSTER_NAME] \
+  --no-enable-autoupgrade \
   --node-taints agones.dev/agones-system=true:NoExecute \
   --node-labels agones.dev/agones-system=true \
   --num-nodes=1
 ```
+{{% /feature %}}
 
 _Optional_: Create a node pool for [Metrics]({{< relref "../Guides/metrics.md" >}}) if you want to monitor the Agones system using Prometheus with Grafana or Stackdriver.
+
+{{% feature expiryVersion="1.1.0" %}}
+```bash
+gcloud container node-pools create agones-metrics \
+  --cluster=[CLUSTER_NAME] \
+  --node-taints agones.dev/agones-metrics=true:NoExecute \
+  --node-labels agones.dev/agones-metrics=true \
+  --num-nodes=1
+```
+{{% /feature %}}
+{{% feature publishVersion="1.1.0" %}}
 ```bash
 gcloud container node-pools create agones-metrics \
   --cluster=[CLUSTER_NAME] \
+  --no-enable-autoupgrade \
```

Comment:
Do we need this here? If grafana/prom restart, is that a bad thing?

Reply:
I'm not sure if it's a bad thing. But if we've disabled it on the primary node pool and the agones system node pool, we might as well do it for this single node as well so that the whole cluster is running one version on all nodes. It also prevents drift where the cluster will end up in a state that we haven't tested (since we only test on a single k8s version).